articles/reactive-combinetimestamp.md

```diff
@@ -4,3 +4,7 @@ title: CombineTimestamp
 ---
 
 ![Marble diagram](~/images/reactive-combinetimestamp.svg)
+
+`CombineTimestamp` is used primarily to create a sequence of <xref href="System.Reactive.Timestamped`1"/> values for downstream operators, when timestamps have been extracted from other sources.
+
+Alternatively, it can be used to preserve the value of a timestamp during post-processing operations. In this case, even though the timestamped value may be transformed multiple times, we can keep the original acquisition timestamp in a branch and use the <xref href="Bonsai.Reactive.Zip"/> operator followed by `CombineTimestamp` to carry the timestamp value forward.
```

Comment:
This one might be nice to include an example. Just because the overload might not be obvious idk...
![image](https://user-images.githubusercontent.com/7049351/209716139-a4501776-be80-47f0-b46e-5a728e662be5.png)